kuma/users/tests/test_views.py

```diff
@@ -206,7 +206,7 @@ def test_profile_edit(self, unsubscribe, subscribe, lookup_user):
         doc = pq(r.content)
 
         eq_(profile.fullname,
-            doc.find('#profile-edit input[name="profile-fullname"]').val())
+            doc.find('#profile-edit input[name="profile-title-fullname"]').val())
```

Comment:
This is [failing on Travis](https://travis-ci.org/mozilla/kuma/builds/45737532) because the test is looking for `name="profile-title-fullname"` but the page has  [`class="profile-title-fullname"`](https://github.com/mozilla/kuma/pull/2972/files#diff-b3f5404653845f826bbf658e10367567R41)?